integration-tests/src/test/java/oracle/weblogic/kubernetes/ItParameterizedDomain.java

```diff
@@ -444,6 +450,172 @@ public void testLivenessProbe() {
         .isTrue();
   }
 
+  @Test
+  @DisplayName("Test dataHome override in a domain with domain in image type")
+  public void testDataHomeOverrideDomainInImage() {
```

Comment:
Move the description to javadoc section for the test method

Reply:
fixed